torch/_dynamo/eval_frame.py

```diff
@@ -789,6 +793,75 @@ def run_node(self, n):
                 r.node.meta["val"] = self.current_node.meta["val"]
             return r
 
+    class AddRuntimeAssertsInInputConstraint(torch.fx.interpreter.Transformer):
+        def __init__(
+            self,
+            m,
+        ):
+            super().__init__(m)
+            self.count = 0
+            self.constraints_id_to_constraint = defaultdict(list)
+            if constraints is not None:
```

Comment:
Is this captured from above? If so then why bother making self.constraints_id_to_constraint, maybe compute it above too...

Reply:
Yeah I can :)

Reply:
Actually left it here because this map is only needed within this pass.